### DISCUSSION TITLE
Location of API Tokens

### DISCUSSION THREAD

**Discussion by madcar86** [SecKatie/ha-wyzeapi#507] [Q&A] [answered]
Can the Key ID and API Key be found anywhere within the config files? I forgot to record the API and would like to record it.

**Comment by c0nstantinator** [comment]
Same question! 
![CleanShot 2024-06-06 at 16 41 53@2x](https://github.com/SecKatie/ha-wyzeapi/assets/128792536/6026db1d-c5b0-4979-b8ec-1e7f8d98778e)

Wyze API website shows KEY ID, but the API key is hidden. And support saying there is no way to view it. They suggest deleting and creating a new one. Which is not a good option.

**Reply by jjroysdon** [reply, marked as answer]
You need to make a note of the Api Key when you generate it. Otherwise, generate a new one (which will automatically delete/expire the old one).
